### ISSUE TITLE
Start Time is changed when End Time is updated

### ISSUE THREAD

**Issue by itshields** [alexjlockwood/ShapeShifter#184]
Repro:

- Create an animation, for example's sake let's say the start time is 400 and the end time is 600 when it's created
- Change the end time to something, for example let's say 300
- Start time is modified to the first digit of the end time (3 in this example) when you start typing 300

I'm assuming this is because the value you're typing in the end time comes before what was originally set in the start time, but you should wait until the user presses enter or the view loses focus to determine this instead of right when typing starts

**Comment by alexjlockwood** [comment]
Yup, known issue. I agree it is super annoying... I get annoyed by it too. Working on it. :)
